README.md

```diff
@@ -44,13 +44,23 @@ See individual [packages](#packages) to learn how to get started.
 - `yarn install`
 - `yarn start`
 
-See [`package.json`](./package.json) for all available tasks.
+See [`package.json`][package.json] for all available tasks.
 
 ## License
 
 See the [LICENSE](LICENSE.md) file for information.
 
 [alma-career]: https://github.com/lmc-eu
+[packages-analytics]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/analytics
```

Comment:
Glad that all links are grouped together at the bottom of the file and they are called via reference/key. 🙌🏻

I was thinking if the rest of the files should have same structure, but that's not explicitly mentioned in the issue description, so... 🤷🏻‍♀️

Reply:
Thank you! <3

Reply:
Refactored😉